### PR TITLE
🔨 Review env-vars for garbage-collector and event-listener services (⚠️ devops)

### PR DESCRIPTION
**Pull request by pcrespov**
## What do these changes do?

The garbage-collector service (`wb-garbage-collector`) and event-listener service (`wb-db-event-listener`) are a "reduced" variants of the `web-server` service where some of plugins are **disabled**.

- PROBLEM: when a new plugin is added to the `web/server` we tend to forget to enable/disable it in the `wb-garbage-collector` and `wb-db-event-listener` variants. Generally it does not have a negative impact but sometimes it might be problematic.
- proposed SOLUTION: I can add a test that ensure all plugin settings are resolved using the env-vars.
- ♻️ Extra fixes on issues reported by @odeimaiz  (thx!)
  - fixes OAS in web-server
  - test was not running in CI because it must be within folders 01, 02 or 03
  
## Related issue/s

- Improving env-vars deployment workflow

## How to test
@YuryHrytsuk @mrnicegyu11  @matusdrobuliak66 please note how we could inspect the different settings for the webserver-based services:
1. create docker-compose for production
```cmd
make .stack-simcore-production.yml
```
2. **inspect environment section** for the following services:  `webserver`, `wb-garbage-collector`, `wb-db-event-listener`
```
code .stack-simcore-production.yml
```
3. Deploy the stack
```cmd
make build
make up-prod
```
4. download settings of apps running with the webserver image  of by running script
```cmd
./download-deployed-webserver-settings.bash 
```
produces `*-settings.json` files.
5. use a **diff tool** to compare the three settings 

## DevOps  ⚠️

- new `WB_GC_WALLETS`,  `WB_GC_INVITATIONS`, `WB_GC_PAYMENTS`
- new `WB_DB_EL_WALLETS`,  `WB_DB_EL_INVITATIONS`, `WB_DB_EL_PAYMENTS`

### CODE REVIEW

File: .env-devel [hunk 1]
```diff
@@ -236,59 +238,78 @@ WB_GC_LOGLEVEL=DEBUG
 WB_GC_GARBAGE_COLLECTOR='{"GARBAGE_COLLECTOR_INTERVAL_S": 30}'
 WB_GC_RESOURCE_MANAGER_RESOURCE_TTL_S=60
 
-WB_GC_ANNOUNCEMENTS=0
+
+#
+# NOTE: PC->Yuri. The env-vars below are to "disable plugins" and therefore must not
+# be changed. We could directly add them in the docker-compose environment section
```
Comment (mrnicegyu11) [inline .env-devel:244]: good suggestion

**Review by mrnicegyu11 — APPROVED** [review]
thx

**Review by odeimaiz — COMMENTED** [review]
Merci!

**Review by matusdrobuliak66 — APPROVED** [review]
nice

**Review by GitHK — APPROVED** [review]

File: scripts/download-deployed-webserver-settings.bash [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+# all containers running on the image `local/webserver:production`
+containers=$(docker ps -a --filter "status=running" --filter "ancestor=local/webserver:production" --format "{{.ID}}")
+
+for container_id in $containers
+do
+    # Get the name of the container
+    container_name=$(docker inspect -f '{{.Name}}' "$container_id" | cut -c 2-)
+
+    # Execute the command in the container to create settings.json
+    docker exec "$container_id" simcore-service-webserver settings --as-json > "${container_name}-settings.ignore.json"
```
Comment (YuryHrytsuk) [inline scripts/download-deployed-webserver-settings.bash:12]: Could you explain what is this for?

**Review by YuryHrytsuk — APPROVED** [review]
Thank you. I reviewed `env.devel`. `WB_GC_*` and `WB_EL_*` have now the same number of settings (`30`). Well done :tada:

**Comment by YuryHrytsuk** [comment]
I am thinking of a test that will compare all the different configs we have (e.g. `env.devel`, aws prod `repo.config`, dalco stag `repo.config`, ...) and fail if any of them have different set of keys

File: scripts/download-deployed-webserver-settings.bash (hunk 2 above)
Comment (pcrespov) [reply scripts/download-deployed-webserver-settings.bash:12]: should be self-explanatopry :-) ... More details in the meeting later today

**Pull request merged**